### ISSUE TITLE
String output is printed with extra quotes

### ISSUE THREAD

**Issue by yoshuawuyts** [lrlna/femme#12]
https://github.com/http-rs/tide/pull/495#discussion_r426254201 shows that command line output is printed with extra quotes. We should fix this.

```json
{"level":30,"time":1589717147052,"msg":"--> Response sent","method":"\"GET\"","status":"200","duration":"\"157ms\"","path":"\"/api/v1/users\""}
```

I believe #10 makes some progress towards this by removing the intermediate strings. But we should still do better. And add tests to ensure we catch this. Thanks!

**Comment by yoshuawuyts** [comment]
Think I found the root of the bug: https://docs.rs/log/0.4.8/src/log/kv/value/internal.rs.html#111-117

Here we see that both the `Debug` and `Display` branches use the same logic -- relying on the Debug output. This means that strings are always printed with extra quotes, which is the source of our problem. We should fix this upstream and write a test for it.

**Comment by yoshuawuyts** [comment]
I don't have the bandwidth today to open an issue against `log`; if someone could open an issue there that'd be great!

**Comment by yoshuawuyts** [comment]
Closed by #13 — not stoked about it, but good enough ^^
